theme/islandora-dublin-core-description.tpl.php

```diff
@@ -15,6 +15,6 @@
 <div class="islandora-metadata-sidebar">
   <?php if (!empty($dc_array['dc:description']['value'])): ?>
     <h2><?php print $dc_array['dc:description']['label']; ?></h2>
-    <p property="description"><?php print $dc_array['dc:description']['value']; ?></p>
+    <p property="description"><?php print nl2br(filter_xss($dc_array['dc:description']['value'])); ?></p>
```

Comment:
I totally agree @adam-vessey. I initially had it there in the preprocess function, but then I took a look at [islandora-dublin-core-display.tpl.php](https://github.com/Islandora/islandora/blob/7.x/theme/islandora-dublin-core-display.tpl.php#L24-L27), and we are doing the filter_xss call there, so I thought the two templates should be the same. 

I'll make an update with it changed in the preprocess function.